pydle/features/ctcp.py

```diff
@@ -21,24 +22,20 @@ async def on_ctcp(self, by, target, what, contents):
         Client subclasses can override on_ctcp_<type> to be called when receiving a message of that specific CTCP type,
         in addition to this callback.
         """
-        pass
```

Comment:
My personal preference is to leave a `...` instead of a `pass` when writing ABCs. `s/pass/...` please.